mrp_project_analytic/models/mrp_production.py

```diff
@@ -40,6 +40,14 @@ def mark_analytic_mrp_from_child(self):
         )
         aml_ids = raw_material_moves.sudo().mapped('account_move_ids.line_ids')
         
+        for move in self.move_raw_ids.filtered(lambda x: x.state == "done"):
+            mr_qty = sum(self.search([('project_id', '=', self.project_id.id), ('state', '=', 'done')]).move_raw_ids.filtered(lambda x: x.product_id == move.product_id.id).mapped('quantity_done'))
+            mt_total = mr_qty + move.product_uom_qty
+            pol_qty = sum(self.env['purchase.order.line'].search([('account_analytic_id', '=', self.project_id.analytic_account_id.id), ('state', 'in', ['purchase', 'done']), ('product_id', '=', move.product_id.id)]).mapped('product_uom_qty'))
```

Comment:
`mrp_project_analytic` doesn't depend on purchase. We should consider adding purchase or `purchase_mrp` as dependency, or hooking this module and creating another one with this piece of code that actually depends on any purchase addon.